### PR TITLE
Adding a pudding recipe

### PR DESCRIPTION
**Pull request by afpearlman**

### CODE REVIEW

File: afpearlman.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Did I actually create a new file?
```
Comment (MBenkert16) [inline afpearlman.md]: yes you did

File: afpearlman.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+Is this good? bad?
```
Comment (MBenkert16) [inline afpearlman.md]: It is bad, there is no recipe.

**Comment by MBenkert16** [comment]
Could you please include a recipe? :smile:

**Comment by jae9970** [comment]
LGTM :+1: Let's :rocket: it! :last_quarter_moon_with_face: Good night

**Pull request merged**